app/graphql/albums/resolvers.js

```diff
@@ -11,6 +12,13 @@ exports.getAlbums = async (_, params) => {
   return tempAlbums.map(album => changeUserIdByArtist(album));
```

Comment:
Just as a reminder you can use _pointfree style_: tempAlbums.map(changeUserIdByArtist)